### PR TITLE
Omr work addition to 2.3

### PR DESCRIPTION
**Pull request by nasehim7**
Porting the changes from imeta to master. Changes made wherever was required to make it compatible. Major Update: Edata.cpp addition which does writing of the metadata for Audiveris. Moreover, Adding Grace Notes, Bracket, Repeat Dots, Bracketed Time Signature and Tuplet Implementation.

### CODE REVIEW

**Comment by nasehim7** [comment]
@Jojo-Schmitz The strange thing is I am able to build and fully work with these commits on my imeta, which is rebased with current 2.3.

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ BUILD_LAME="ON" # Non-free, required for MP3 support. Override with "OFF" to dis
 BUILD_PULSEAUDIO="ON" # Override with "OFF" to disable.
 BUILD_JACK="ON"       # Override with "OFF" to disable.
 BUILD_PORTAUDIO="ON"  # Override with "OFF" to disable.
-BUILD_WEBKIT="ON"     # Override with "OFF" to disable.
+BUILD_WEBKIT="OFF”     # Override with "ON” to enable.
```
Comment (Jojo-Schmitz) [inline Makefile]: Wrong " used here, leads to Travis CI fail "Untermitated string"

**Comment by Jojo-Schmitz** [comment]
`error: ‘Format_Grayscale8’ is not a member of ‘QImage’` due to Travis (and AppVeyor) using Qt 5.4

**Comment by anatoly-os** [comment]
@nasehim7 I stopped the build and will restart it once we fix master branch. I'll keep you updated.

**Comment by nasehim7** [comment]
@anatoly-os Sure no problem :)

**Comment by anatoly-os** [comment]
Restarted

**Comment by nasehim7** [comment]
Thanks :D

**Comment by nasehim7** [comment]
Still failed at GrayScale8 on AppVeyor. It's building fine on my imeta rebased with 2.3.2. Need more guidance on this. @Jojo-Schmitz @anatoly-os @lasconic

**Pull request closed without merging**